oxidation/crates/parsec_api_protocol/tests/test_transport.rs

```diff
@@ -0,0 +1,33 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use async_std::net::{TcpListener, TcpStream};
+use async_std::task;
+use parsec_api_protocol::Transport;
+use rstest::rstest;
+
+#[rstest]
+fn test_transport() {
+    smol::block_on(async {
```

Comment:
Now use Parsec handshake instead of `b"foobar"` payloads ;-)

Reply:
I can't while handshake is not merged :grimacing:

Reply:
Done :+1: